imports/plugins/core/checkout/server/methods/workflow.js

```diff
@@ -335,6 +335,39 @@ Meteor.methods({
     check(order, Object);
     check(itemIds, Array);
 
+    if (status === "coreOrderItemWorkflow/packed") {
+      const shopId = Reaction.getShopId();
+
+      const existingOrder = Orders.find({ _id: order._id }).fetch();
```

Comment:
You want to just do `findOne` there.

Reply:
Yes, I needed just a specific order.

Reply:
Right so just `const existingOrder = Orders.findOne({ _id: order._id })` which eliminates the need for the `existingOrder[0]`

Reply:
Wow, cool.